src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,20 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    private Excavator firstExcavator = new Excavator("First Excavator", "black", 2000);

```

Comment:
Consider making the Excavator instances (firstExcavator, secondExcavator, thirdExcavator) constants (final) if they are meant to be fixed values.